### PR TITLE
refactor: clean up new-price-list feature flag

### PR DESCRIPTION
**Pull request by rajsam003**
The type of this PR is: Refactor

This PR solves [FX-4670]

### TODO :

- [ ] Archive the feature flag from unleash after this PR gets deployed

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FX-4670]: https://artsyproduct.atlassian.net/browse/FX-4670?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx [hunk 1]
```diff
@@ -214,11 +213,5 @@ export const PriceRangeFilterOld: FC<PriceRangeFilterProps> = ({
...
 
 export const PriceRangeFilter: FC<PriceRangeFilterProps> = props => {
-  const isNewPriceFilterEnabled = useFeatureFlag("new_price_filter")
-
-  if (isNewPriceFilterEnabled) {
-    return <PriceRangeFilterNew {...props} />
-  }
-
-  return <PriceRangeFilterOld {...props} />
+  return <PriceRangeFilterNew {...props} />
```
[the remark this answers is not in the document]
Comment (dimatretyak) [reply src/Components/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx:216]: good suggestions!

**Review by dimatretyak — DISMISSED** [review]
LGTM 👍

File: src/Components/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx [hunk 2]
```diff
@@ -1,224 +1,223 @@
...
-  { name: "$50K+", value: "50000-*" },
-  { name: "$25K – $50K", value: "25000-50000" },
-  { name: "$10K – $25K", value: "10000-25000" },
-  { name: "$5K – $10K", value: "5000-10000" },
-  { name: "$1K – $5K", value: "1000-5000" },
-  { name: "$0 – $1,000", value: "0-1000" },
-]
+import { useFilterLabelCountByKey } from "Components/ArtworkFilter/Utils/useFilterLabelCountByKey"
+import { useMemo, FormEvent } from "react"
+import { Box, Text, Range } from "@artsy/palette"
```
Comment (anandaroop) [inline src/Components/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx:13]: Combine these imports with L1–L2?

**Review by anandaroop — DISMISSED** [review]
Looking good, thanks @rajsam003!

**Review by dblandin — DISMISSED** [review]
LGTM! Thanks for cleaning this up!!

File: src/Components/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx (hunk 2 above)
Comment (rajsam003) [reply src/Components/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx:13]: Oh yes.. Why didn't it complain 🤔?

**Review by gkartalis — APPROVED** [review]

**Pull request merged**